master/internal/db/postgres_rp_workspace_bindings_intg_test.go

```diff
@@ -83,16 +84,92 @@ func TestAddAndRemoveBindings(t *testing.T) {
 
 func TestBindingFail(t *testing.T) {
 	// Test add the same binding multiple times - should fail
+	ctx := context.Background()
+	pgDB := MustResolveTestPostgres(t)
+	MustMigrateTestPostgres(t, pgDB, MigrationsFromDB)
+
+	user := RequireMockUser(t, pgDB)
+	workspaceIDs, err := MockWorkspaces([]string{"test1", "test2", "test3"}, user.ID)
+	require.NoError(t, err, "failed creating workspaces: %t", err)
+	defer func() {
+		err = CleanupMockWorkspace(workspaceIDs)
+		if err != nil {
+			log.Errorf("error when cleaning up mock workspaces")
+		}
+	}()
```

Comment:
Good stuff! I will use this in the tests I write. So I don't need to manually reset my db when the test fails.